diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7b13753..b62b849 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -474,9 +474,9 @@ macro(common_libktx_settings target enable_write library_type)
 
     )
     if(IOS)
-        set_target_properties(${target} PROPERTIES
-            FRAMEWORK TRUE
-        )
+#        set_target_properties(${target} PROPERTIES
+#            FRAMEWORK TRUE
+#        )
     endif()
 
     if( NOT ${library_type} STREQUAL STATIC )
